test/e2e/unifiedpush_test.go

```diff
@@ -78,7 +69,7 @@ func UnifiedpushTest(t *testing.T) {
 	}
 
 	// Ensure UPS was deleted successfully
-	if err := waitForDeploymentConfig(t, *dcV1Client, namespace, unifiedPushServerName, 0); err != nil {
+	if err := e2eutil.WaitForDeployment(t, f.KubeClient, namespace, unifiedPushServerName, 0, retryInterval, timeout); err != nil {
```

Comment:
The test is failing because the method [`WaitForDeployment`](https://github.com/operator-framework/operator-sdk/blob/master/pkg/test/e2eutil/wait_util.go#L46-L72) doesn't support checking for 0 replicas unlike the [helper function for DC in this test](https://github.com/aerogear/unifiedpush-operator/pull/74/files#diff-74fafcf65f86b587f5202b55bf1edc80L142).
We can either reimplement the function that supports checking for 0 replicas in this test, or just delete the check.

Reply:
wanted to get rid of the custom helper function for DC but maybe need to introduce it back.
thanks for the heads up @psturc